.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
```

Comment:
You could add a comment clarifying that this file is intended to be used primarily by Travis CI, but I won't force you to.

Reply:
I decided against doing this because the name of the file has "travis" in it!